### PR TITLE
Update to latest googleapis

### PR DESCRIPTION
**Pull request by jbolinger**

### CODE REVIEW

File: googleapis-common-protos-types/googleapis-common-protos-types.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 Gem::Specification.new do |spec|
   spec.name          = "googleapis-common-protos-types"
-  spec.version       = "1.0.3"
+  spec.version       = "1.1.0"
```
Comment (blowmage) [inline googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: The grpc gem declares a dependency on this gem, and the dependency is `~> 1.0.0`. So this means that `1.1.0` is out of scope for what can be used with the grpc gem, and this release won't be used.

Comment (jbolinger) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: where did you see this? (I only see the `~> 1.0` dependency)

Comment (blowmage) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: https://rubygems.org/gems/grpc/versions/1.19.0

Comment (blowmage) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: https://github.com/grpc/grpc/blob/master/grpc.gemspec

Comment (jbolinger) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: oh, *the* grpc gem. That's good to know, I had no idea.

Comment (blowmage) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: I wonder if they intend for the dependency to be that narrowly defined.

Comment (jbolinger) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: I have no idea. You think it's worth sending a PR to loosen it?

Comment (blowmage) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: Yeah, it wouldn't hurt. If anything, it should provide a place for them to explain why the dependency is so narrowly defined.

Comment (jbolinger) [reply googleapis-common-protos-types/googleapis-common-protos-types.gemspec]: https://github.com/grpc/grpc/pull/18635

**Pull request closed without merging**